pkg/report/testdata/linux/report/722

```diff
@@ -0,0 +1 @@
+xfs: Unknown parameter '                                                                Booting the kernel.                                                                                                                                             '
```

Comment:
These tests should be full console output including time/context `[   65.924552][ T6734] ` b/c that's what parsed by pkg/report.